### ISSUE TITLE
Ability to filter Peers Strength (and other pages) by Change author

### ISSUE THREAD

**Issue by Cireo** [change-metrics/monocle#1037]
I work with a repository that has a fair amount of bot-authored PRs (primarily for backports), and I'd like to be able to omit them from results.  Right now `"author:"` refers to "Peer reviewer" on that page, and I couldn't find any way to refer to the "Change author"

**Comment by morucci** [comment]
Using "not authors: bot1 and not author:bot2" should work as expected for all metrics. Also note that a search alias could be used to group all bot idents.

However yes for the Peers Strength metric the author query field filters the "Peer reviewer" in the UI. There is no way in the query language to change this behavior. 

I'll take time to review how this is computed and what could be done to improve usage for your use case.

Thanks for the report.

**Comment by Cireo** [comment]
Yes, I've greatly appreciated the search aliases, adding `not bots` is quite convenient.  Thanks for thinking about the peer reviewer case! 🙇
